autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -452,6 +452,23 @@ def leaderboard(self, X=None, y=None, silent=False):
                 print(leaderboard)
         return leaderboard
 
+    # TODO: Should this return a dictionary or pandas Series? Current it returns a Series
```

Comment:
I think sticking with pandas formats for things like leaderboard, feature importance, ensemble model weights is a nice way to go.  But for nested info like from HPO, we should stick with dicts.

Reply:
Sounds good